modules/gittar/pkg/gc/gc.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// This program is free software: you can use, redistribute, and/or modify
+// it under the terms of the GNU Affero General Public License, version 3
+// or later ("AGPL"), as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+package gc
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os/exec"
+	"strings"
+
+	"github.com/erda-project/erda/modules/gittar/pkg/limit_sync_group"
+	"github.com/erda-project/erda/pkg/cron"
+	"github.com/sirupsen/logrus"
+)
+
+// all gittar repository root address
+const repositoryRootAddr = "/repository"
```

Comment:
Get repositoryRootAddress from config.
If someone changes the root addr one day, it is easy for him to be unclear about the gc configuration, leading to a mismatch.